diagnostics/model_test/verify_model.py

```diff
@@ -208,12 +252,19 @@ def parse_json():
         input_tensor_dim_map[name] = random_input
 
     # Run the model on reference backend
-    result_tf_graph_arrs, out_tensor_names_cpu = calculate_output(
+    result_tf_graph_arrs, out_tensor_names_cpu, tf_skipped_tensors = calculate_output(
         parameters, device1, input_tensor_dim_map)
     # Run the model on testing backend
-    result_ngraph_arrs, out_tensor_names_ngraph = calculate_output(
+    result_ngraph_arrs, out_tensor_names_ngraph, ngraph_skipped_tensors = calculate_output(
         parameters, device2, input_tensor_dim_map)
 
+    assert all(
+        [i == j for i, j in zip(tf_skipped_tensors, ngraph_skipped_tensors)])
```

Comment:
Zipping two lists will zip upto the shorter list. So we might need something like:
```
assert (len(tf_skipped_tensors) == len(ngraph_skipped_tensors)) and all(
        [i == j for i, j in zip(tf_skipped_tensors, ngraph_skipped_tensors)])
```